src/store/user/actions.ts

```diff
@@ -0,0 +1,25 @@
+import { ActionTree } from 'vuex';
+import { RootStateInterface } from '@/store/types';
+import { UserStateInterface, UserActionTypes, UserMutationTypes, Actions } from '@/store/user/types';
+import { HTTP } from '@/core/api/http-common';
+
+export const actions: ActionTree<UserStateInterface, RootStateInterface> & Actions = {
+  [UserActionTypes.SET_USER_INFO]: async ({ commit }, userData) => {
+    const bodyData = {
+      registration_token: userData?.registrationToken,
```

Comment:
why all props optional (undefined)
according to flow, userData have required props and can't be undefined